go.mod

```diff
@@ -24,29 +24,29 @@ require (
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hdevalence/ed25519consensus v0.0.0-20210204194344-59a8610d2b87
 	github.com/improbable-eng/grpc-web v0.14.1
-	github.com/jhump/protoreflect v1.9.0
-	github.com/magiconair/properties v1.8.5
+	github.com/jhump/protoreflect v1.12.1-0.20220417024638-438db461d753
```

Comment:
hmm this needs to be downgraded but it affects tendermint

Reply:
Can you elaborate on "affects Tendermint"?

Reply:
when I downgrade to 1.12.0 tendermint downgrades and vice versa. On main we have this version and the tests pass. Not sure what's up

Reply:
We might have to force the `protoreflect` version via a `replace` directive in order to use TM v0.34.20

Reply:
this is super weird. I cant get it work even with the old versions